src/test/resources/features/Exclusions.feature

```diff
@@ -0,0 +1,70 @@
+@Registration
+
+Feature: OSS Exclusion journeys
+
+  Scenario: Kickout in the registration journey when the user is quarantined on the One Stop Shop service
+    Then the user accesses the authority wizard
+    And the user signs into authority wizard as an Organisation Admin with OSS and VAT enrolment 100000025
+    Then the user answers no on the ioss-registered page
+    And the user answers yes on the selling-goods-outside-single-market page
+    And the user answers yes on the goods-value page
+    And the user answers yes on the registered-for-vat-in-uk page
+    And the user answers yes on the ni-based page
+    And the user continues through the register-to-use-service page
+    Then the user is on the scheme-quarantined page
```

Comment:
since https://github.com/hmrc/ioss-registration-frontend/pull/148#discussion_r1822894782 will hold off on approving/merging this until the correct page is implemented

Reply:
New page has been implemented and tests updated